desc/magnetic_fields.py

```diff
@@ -367,134 +497,162 @@ def save_BNORM_file(
         return None
 
 
-class ScaledMagneticField(_MagneticField):
-    """Magnetic field scaled by a scalar value.
+class CurrentPotentialField(_MagneticField, FourierRZToroidalSurface):
```

Comment:
what changes were actually made here?

Reply:
All I did was alphabetize the classes/functions in this file. I reverted that change now so the diff is readable.